### PR TITLE
Bluetooth: controller: fixing CI complaints on up-to-main PR

### PR DESCRIPTION
**Pull request by erbr-ot**
This should fix most complaints from CI on the upstream to main PR

Signed-off-by: Erik Brockhoff <erbr@oticon.com>

### CODE REVIEW

**Review by sjanc — APPROVED** [review]

**Comment by erbr-ot** [comment]
Fixed the 'last' three warning.

**Review by kruithofa — APPROVED** [review]
LGTM

**Comment by kruithofa** [comment]
On 2nd thoughts, there are still a couple of lines that are over 100 chars :(

File: tests/bluetooth/controller/common/src/helper_util.c [hunk 1]
```diff
@@ -293,14 +293,15 @@ uint16_t event_counter(struct ll_conn *conn)
...
+	 * otherwise return the next event counter value
+	 */
 	if (event_active)
 		event_counter--;
 
 	return event_counter;
 }
 
-void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```
Comment (kruithofa) [inline tests/bluetooth/controller/common/src/helper_util.c:304]: line is now over 100chars
```suggestion
void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```

File: tests/bluetooth/controller/common/src/helper_util.c [hunk 2]
```diff
@@ -321,7 +322,7 @@ void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, str
 	sys_slist_append(&lt_tx_q, (sys_snode_t *)rx);
 }
 
-void lt_rx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```
Comment (kruithofa) [inline tests/bluetooth/controller/common/src/helper_util.c:325]: line is now over 100 chars
```suggestion
void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```

File: tests/bluetooth/controller/common/include/helper_util.h [hunk 3]
```diff
@@ -25,13 +25,13 @@ uint16_t event_counter(struct ll_conn *conn);
 	ut_rx_node_real(__FILE__, __LINE__, _opcode, _ntf_ref, _param)
 #define ut_rx_q_is_empty() ut_rx_q_is_empty_real(__FILE__, __LINE__)
 
-void lt_tx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```
Comment (kruithofa) [inline tests/bluetooth/controller/common/include/helper_util.h:28]: line is now over 100chars
```suggestion
void lt_tx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                       struct ll_conn *conn,
```

File: tests/bluetooth/controller/common/include/helper_util.h [hunk 4]
```diff
@@ -25,13 +25,13 @@ uint16_t event_counter(struct ll_conn *conn);
...
 		void *param);
-void lt_rx_real(const char *file, uint32_t line, helper_pdu_opcode_t opcode, struct ll_conn *conn,
+void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, struct ll_conn *conn,
```
Comment (kruithofa) [inline tests/bluetooth/controller/common/include/helper_util.h:30]: line is now over 100chars
```suggestion
void lt_rx_real(const char *file, uint32_t line, enum helper_pdu_opcode opcode, 
                        struct ll_conn *conn,
```

**Review by kruithofa — CHANGES_REQUESTED** [review]

File: tests/bluetooth/controller/common/include/helper_util.h (hunk 4 above)
Comment (erbr-ot) [reply tests/bluetooth/controller/common/include/helper_util.h:30]: Sorry! So much for that help :-)

File: tests/bluetooth/controller/common/src/helper_util.c (hunk 1 above)
Comment (erbr-ot) [reply tests/bluetooth/controller/common/src/helper_util.c:304]: Doh!

**Review by kruithofa — APPROVED** [review]

**Review by thoh-ot — APPROVED** [review]

**Pull request merged**